### ISSUE TITLE
Enchanted Armoir item disappears upon equipping it

### ISSUE THREAD

**Issue by otomatonom** [HabitRPG/habitica#7489]
General Info
- UUID: 61d9aab8-196f-4f52-90d1-c94134249500
- Browser: Chrome
- OS: OS X 10.9.5
### Description

I clicked to purchase an Enchanted Armoir and was awarded something called a "Red Crow" or something that looked like a red-and-black shepherd's crook (I didn't get the best look at the name that will be explained later). I go to my inventory and am looking at my different items, and decide to try on the item I just got because it had good perception and strength. When I clicked on it, a red textbox appeared (that looked the same otherwise as the textbox when your receive new items or complete a task) that said something like "error___, item not found" (I clicked away instead of taking a screenshot unfortunately). The textbox looked especially like when you complete a task because it was paired with other boxes for minus-gold (-3) and minus-experience, as if I had undone one of my completed tasks or dailies. After the textboxes went away, the item disappeared. By the end of it I no longer had the item and was down 103 gold. On the suggestion of the issue-reporting guidelines, I tried closing the tab and reloading it, but that did not result in the return of the gold or the item. The console error screenshot I took was after I had done this, so I not sure how much help it will be.
#### Console Errors

<img width="551" alt="screen shot 2016-05-24 at 7 01 47 pm" src="https://cloud.githubusercontent.com/assets/5776362/15527242/7474f728-21ed-11e6-97a4-c55ac4205b6c.png">

**Comment by ninetyd** [comment]
I've just had the same issue. Got a Bat Wand from the Armoire, went to equip it as part of my Costume, which seemed to work as I could see it on my avatar picture but then the item disappeared somehow and I was back to being equipped with my previous item, the Barrister Gavel (although I'm not sure it showed up on my avatar). I reloaded the page but the item didn't reappear and gold was not refunded. I don't know if I also got an error message upon clicking on the wand.

```
UUID: 97e0905f-cafa-44bd-848d-4816b24b5b4b
Browser: Firefox 46.0.1
OS: Windows 10 version 1511
```
